sherpa/sim/__init__.py

```diff
@@ -785,11 +785,21 @@ class ReSampleData(NoNewAttributesAfterInit):
 
     Returns
     -------
-    The class returns the best fit parameters for each realization.
+    sampled : dict
+       The keys are samples, which contains the resampled data
+       used in the fits as a niter by ndata array, and the free
+       parameters in the fit, containing a NumPy array containing
```

Comment:
Maybe add 'best-fit' parameters in the fit?

Reply:
I've just made an update. It is now

https://github.com/sherpa/sherpa/blob/c3de99e5136f75a0e6829942c6c94887a8fdff87/sherpa/sim/__init__.py#L788-L793